### PR TITLE
Custom resource ids

### PR DESCRIPTION
**Pull request by leosimas**
Now we can use custom resources for dialog texts: title, message, etc.
Created the DialogOptions for a better dialog creation.

### CODE REVIEW

**Comment by hotchemi** [comment]
Hi, thx for your PR.
I think it already provides the way to change each text. Why do you need new way to change these?
https://github.com/hotchemi/Android-Rate#custom-dialog

**Comment by leosimas** [comment]
An app that has only one language, for example: pt.
The project only got the values folder with portuguese strings.
My app should be always in Portuguese cause most user generated content is in Portuguese and even if the user likes to have Android in English.
It is strange to have an app in English but most of its content is in another language.

So, if I use this library I could get the dialog in another language because of the values-something folder. But I would like to show it just in Portuguese for example.

And its just optional, already using the default "rate_" resource ids.

File: library/src/main/java/hotchemi/android/rate/AppRate.java [hunk 1]
```diff
@@ -69,8 +64,8 @@ public AppRate setRemindInterval(int remindInterval) {
         return this;
     }
 
-    public AppRate setShowNeutralButton(boolean isShowNeutralButton) {
-        this.isShowNeutralButton = isShowNeutralButton;
+    public AppRate setShowLaterButton(boolean isShowNeutralButton) {
+        options.setShowNeutralButton( isShowNeutralButton );
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/AppRate.java]: Can you fix formatter setting to the default of android studio? I don't like it.

**Comment by hotchemi** [comment]
Hi, sorry for the late. 
Ailright, I'm gonna review the code.

File: library/src/main/java/hotchemi/android/rate/DialogOptions.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
+package hotchemi.android.rate;
+
+import android.view.View;
+
+/**
+ * Created by Leonardo on 06/07/2015.
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/DialogOptions.java]: Please delete javadoc.

File: library/src/main/java/hotchemi/android/rate/DialogOptions.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+ */
+class DialogOptions {
```
Comment (hotchemi) [inline library/src/main/java/hotchemi/android/rate/DialogOptions.java]: This class should be final.

**Comment by hotchemi** [comment]
Thx:)

**Comment by leosimas** [comment]
Glad to help :smile: 
Just waiting for the new release

**Comment by hotchemi** [comment]
I'll do that soon:)

**Pull request merged**